uaclient/security.py

```diff
@@ -562,6 +563,40 @@ def fix_security_issue_id(
     }
 
     if "CVE" in issue_id:
+        # Check livepatch status for CVE in fixes before checking CVE api
+        try:
```

Comment:
@lucasmoura is right. If livepatch fails, the variable `status_stdout` will not exist.
You need to create it before the `try` block`.

```suggestion
        status_stdout = None
        try:
```

Easily reproducible:
- Launch an LXD container, don't enable livepatch
- install the client from this branch
- run `ua fix cve-2013-1798`

I'd suggest a test covering this case.

Reply:
@renanrodrigo ha! we commented the same thing at the same time :upside_down_face: 

@dixonmyles the extra unit test for this case is optional, since this is already caught by our integration tests.

Reply:
Thanks, Renan! @orndorffgrant should I get that additional test case created before this gets approved?

Reply:
Great, in that case I’ll make that change to add the variable declaration outside of the try block then. Unfortunately I kinda nuked my Linux install when expanding my root partition (to get enough disk space for multipass to work 🙃), so in the interest of not making you all wait on me fixing that I’ll leave the new test case out